core/examples/http_bench_bin_ops.rs

```diff
@@ -94,8 +153,9 @@ fn op_listen(
   debug!("listen");
   let addr = "127.0.0.1:4544".parse::<SocketAddr>().unwrap();
   let std_listener = std::net::TcpListener::bind(&addr)?;
-  let listener = TcpListener::from_std(std_listener)?;
-  let rid = state.resource_table.add("tcpListener", Box::new(listener));
+  std_listener.set_nonblocking(true)?;
```

Comment:
Is there a reason to create `std::net:TcpListener` instead of using tokio's?

Reply:
Not really, but that's not a change introduced by this PR. 
I believe this "hack" was originally added because Tokio's `bind()` method is async.